### PR TITLE
Features/twitchchateventtype

### PR DESCRIPTION
**Pull request by PythooonUser**
## Summary
Adds the TwitchChatEventType class (see #50) and puts the TwitchChatColor (name change!) in the twitchobserver module (for simplicity).

Updates references to use the appropiate TwitchChatEventType.

Updates the documentation.

Updates version numbers.

### CODE REVIEW

File: docs/conf.py [hunk 1]
```diff
@@ -61,9 +61,9 @@
 # built documents.
 #
 # The short X.Y version.
-version = '0.5.0'
+version = '0.7.7'
```
Comment (joshuaskelly) [inline docs/conf.py]: Could we just import the version from the twitch observer package? Similar to how setup.py works?

```python
from twitchobserver import __version__
```

File: docs/conf.py [hunk 2]
```diff
@@ -61,9 +61,9 @@
...
 # The full version, including alpha/beta/rc tags.
-release = '0.5.0'
+release = '0.7.7'
```
Comment (joshuaskelly) [inline docs/conf.py]: +1 for keeping this version in lock-step with the package version.

File: docs/conf.py (hunk 1 above)
Comment (PythooonUser) [inline docs/conf.py]: Yes, very good. Didn't thought of that.

**Comment by joshuaskelly** [comment]
Feel free to merge at your convenience!

**Pull request merged**